voltdb/src/main/java/org/voltdbycsb/procs/package-info.java

```diff
@@ -0,0 +1,23 @@
+/**
+ * Copyright (c) 2015 YCSB contributors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you
+ * may not use this file except in compliance with the License. You
+ * may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+ * implied. See the License for the specific language governing
+ * permissions and limitations under the License. See accompanying
+ * LICENSE file.
+ */
+
+/**
+ * VoltDB org.voltdbycsb.procs for Put and Scan.  Other org.voltdbycsb.procs
+ * are defined using DDL in ycsb_ddl.sql
+ */
+package org.voltdbycsb.procs;
```

Comment:
this should be in `com.yahoo.ycsb.db.voltdb` or a package under that

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .

Reply:
Should be fixed in 0a534e8434d7e854901fbcf825df0403cc985ba3 .